### ISSUE TITLE
Choosing a Ground Station

### ISSUE THREAD

**Issue by auturgy** [ArduPilot/ardupilot_wiki#536]
[Choosing a Ground Station](http://ardupilot.org/copter/docs/common-choosing-a-ground-station.html) - Fightingwalrus.com is a dead link

**Comment by hamishwillee** [comment]
Thank you. True. 

@rmackay9 My temptation is to remove this entry. Their site is down (all the time). The app hasn't had an update for a year. Thoughts?

**Comment by CraigElder** [comment]
I suggest we can remove it. Fighting Walrus is gone.

**Comment by rmackay9** [comment]
fine with me.

**Comment by hamishwillee** [comment]
Thank you. This has been removed.

**Comment by auturgy** [comment]
No worries, thanks Hamish.
There are a number of other things spread throughout the wiki's that have been overtaken by time. I'm more than happy to help out reviewing and updating if you could let me know the process to do so.
Regards,

James

**Comment by hamishwillee** [comment]
Hi James

That would be excellent! The process is pretty simple make a PR and we'll get reviewed. The PR should explain your reasoning in the same way this issue did.

The editing guide explains most of what you might need to know: http://ardupilot.org/copter/docs/common-wiki_editing_guide.html

IF you get stuck, you can always ask for help here or on the discussion boards.

**Comment by skinnyb8** [comment]
I Really Appreciate your responses to me… it’s the first offer of help I’ve received in four years of trying to accomplish this…

 

First thing that happened is the APM I ordered was a much older configuration and no instructions came with it… I promptly smoked the first one but re ordered and spent a year trying to get it running.

But Now I can’t use it and have over $300 tied up in it… haven’t touched it since last year.

 

It said I could us version 3.0 and it did work perfectly for a little over an hour before the Mission Planner Software automatically updated…  

 

I sent email to several of the You tube Subscriptions that I watched to finally get it running… but nobody can tell me where to get the older version of Mission Planner and turn of Automatic update.

 

I’m working with Arduino on other small electronic “Projects”… but have all but given up on the APM 2.8?? (I think that’s right)

 

Again, I really appreciate being able to “talk” to someone about this…

 

Kenny B
